runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -1760,8 +1760,8 @@ CISCTransform2FindBytes(TR_CISCTransformer *trans)
       //   PassThrough
       //      baseNode
       //
-      ///TR_CISCNode *nullNode = listT->getListHead()->getData();
-      ///TR::Node *nullRepNode = nullNode->getHeadOfTrNodeInfo()->_node;
+      /*TR_CISCNode *nullNode = listT->getListHead()->getData();*/
+      /*TR::Node *nullRepNode = nullNode->getHeadOfTrNodeInfo()->_node;*/
```

Comment:
These two lines continue several lines of comments that preceded them, so I would recommend using the same style of comments for them.  May I ask you to change these two lines to the following?

```
      // TR_CISCNode *nullNode = listT->getListHead()->getData();
      // TR::Node *nullRepNode = nullNode->getHeadOfTrNodeInfo()->_node;
```